src/_data/data.js

```diff
@@ -478,14 +483,36 @@ function crossReferenceFaqsAndGuidanceRequests(faqs, guidanceRequests) {
   });
 };
 
-// Link lists with their FAQs (bidirectional)
+// Link lists with their FAQs and sublists (bidirectional)
 function crossReferenceListsAndFaqs(lists, faqs) {
   lists.forEach(list => {
-    list.faqs = list.faqs.map(faqId => {
-      const faqObject = faqs.find(faq => faq.id === faqId);
-      faqObject.relatedLists.push(list);
-      return faqObject;
-    });
+    list.items = list.itemRefs.map(itemRef => {
+      // First check if it's a list reference
+      const sublist = lists.find(l => l.id === itemRef);
+      if (sublist) {
+        sublist.parentLists.push(list); // Track parent list
+        return {
+          type: 'list',
+          data: sublist
+        };
+      }
+
+      // Otherwise treat as FAQ reference
+      const faqObject = faqs.find(faq => faq.id === itemRef);
+      if (faqObject) {
+        faqObject.relatedLists.push(list); // Track parent list
+        return {
+          type: 'faq',
+          data: faqObject
+        };
+      }
+
+      return null;
+    }).filter(item => item !== null);
+
+    // Calculate counts for display
+    list.faqCount = list.items.filter(item => item.type === 'faq').length;
+    list.listCount = list.items.filter(item => item.type === 'list').length;
```

Comment:
I'm unsure that we should differentiate as to whether children are FAQs or lists. Isn't the type field there for that?

Reply:
As discussed, we should have a faqCount that includes the Faqs of sublists.